starter_code/css/desktop.css

```diff
@@ -0,0 +1,237 @@
+@media(min-width: 1280px){
+
+
+
+body {
+  color: #f0efef;
+  font-size: 26px;
+  line-height: 1.4;
+  color: #606161;
+  font-family: 'Open Sans', Helvetica, Arial, sans-serif;
+  font-weight: 300;
+}
```

Comment:
You should only include the styles here that are _changing_. If the `color`, `font-family` etc aren't changed from smaller screen sizes, there's no need to include them here.